frontend/packages/kubevirt-plugin/src/utils/validations/template/template-validations.ts

```diff
@@ -73,6 +73,14 @@ export class TemplateValidations {
     });
   };
 
+  getDefaultBus = (defaultBus = DiskBus.VIRTIO): DiskBus => {
+    const allowedBuses = this.getAllowedBuses();
+    if (allowedBuses.size === 0) {
```

Comment:
not sure if we should keep it here or just return the defaultBus here for the function consistency. As this branch should never execute